modules/Bio/EnsEMBL/Compara/DBSQL/DnaFragAdaptor.pm

```diff
@@ -207,13 +207,6 @@ sub fetch_all_by_GenomeDB_region {  # DEPRECATED
   Arg [-CODON_TABLE_ID] (opt)
              : Integer. Only returns DnaFrags that are translated with this codon table
   Example    : $human_scaffolds = $dnafrag_adaptor->fetch_all_by_GenomeDB($human_gdb, -COORD_SYSTEM_NAME => 'scaffold', -CELLULAR_COMPONENT => 'NUC', );
-  Arg [2]    : (optional) string $coord_system_name
-  Arg [3]    : (optional) string $name
-  Arg [4]    : (optional) boolean $is_reference
-  Arg [5]    : (optional) string $cellular_component
-  Example    : my $human_chr_dnafrags = $dnafrag_adaptor->
-                   fetch_all_by_GenomeDB_region(
-                     $human_genome_db, 'chromosome')
```

Comment:
This is a leftover of the previous method, `fetch_all_by_GenomeDB_region`, which, BTW was listed as to be removed in e98. Can you remove it in this PR too ? And update DEPRECATED.md at the same time

Reply:
Done. I have also found a `subtest` for the deprecated method and I have removed it.